QuickTests/FunctionalTests/DefinitionsTests.swift

```diff
@@ -0,0 +1,54 @@
+import XCTest
+import Quick
+import Nimble
+
+private var fetches = [Int]()
+
+class FunctionalTests_DefinitionsSpec: QuickSpec {
+    override func spec() {
+        define("one") { return 1 }
```

Comment:
Interesting. You could probably make this even shorter:

```swift
define("one") { 1 }
```

I bet you could also have `define` take an `@autoclosure(escaping)` parameter, which would allow you to write:

```swift
define("one", 1)
```